sync/src/synchronizer/block_fetcher.rs

```diff
@@ -40,23 +36,11 @@ where
             total_difficulty,
         }
     }
-    pub fn initial_and_check_inflight(&self) -> bool {
-        let mut blocks_inflight = self.synchronizer.peers.blocks_inflight.write();
-        let inflight = blocks_inflight
-            .entry(self.peer)
-            .or_insert_with(Default::default);
-
-        if inflight.timestamp < unix_time_as_millis().saturating_sub(BLOCK_DOWNLOAD_TIMEOUT) {
-            debug!(
-                target: "sync", "[block downloader] inflight block download inflight.timestamp {} now {}",
-                inflight.timestamp,
-                unix_time_as_millis()
-            );
-            inflight.clear();
-        }
+    pub fn check_inflight(&self) -> bool {
```

Comment:
The name is bad. I was really surprised when I first saw the function name and it returns true when the peer reaches the limit. Let's make it more verbose, such as `reached_inflight_limit`.